Fix eslint issues.

```bash
➜  localgov git:(main) ✗ ddev eslint-js-fix web/modules/contrib/localgov_step_by_step

/var/www/html/web/modules/contrib/localgov_step_by_step/js/step-by-step-nav.js
    6:2   warning  Unexpected unnamed function                 func-names
    6:23  warning  'drupalSettings' is defined but never used  no-unused-vars
   15:5   error    Expected a default case                     default-case
   17:23  warning  Unexpected unnamed function                 func-names
   42:23  warning  Unexpected unnamed function                 func-names
   96:32  warning  Unexpected unnamed function                 func-names
  108:33  warning  Unexpected unnamed function                 func-names
  120:31  warning  Unexpected unnamed function                 func-names

✖ 8 problems (1 error, 7 warnings)
```

--- a/js/step-by-step-nav.js
+++ b/js/step-by-step-nav.js
@@ -3,94 +3,143 @@
  * Additional behaviour for the Step by step navigation.
  */
 
-(function($, Drupal, drupalSettings) {
+(function lgdStepByStepScript($, Drupal) {
+  Drupal.behaviors.lgdStepByStepNav = {
+    attach(context) {
+      const stepByStep = {};
+      stepByStep.showAllText = 'Show summaries';
+      stepByStep.hideAllText = 'Hide summaries';
+      stepByStep.showStepText = 'Show step summary';
+      stepByStep.hideStepText = 'Hide step summary';
 
-  const stepByStep = {};
-  stepByStep.showAllText = 'Show summaries';
-  stepByStep.hideAllText = 'Hide summaries';
-  stepByStep.showStepText = 'Show step summary';
-  stepByStep.hideStepText = 'Hide step summary';
+      // Set visibility based on specified button.step-show elements.
+      function summaryVisiblity(elements, cmd) {
+        switch (cmd) {
+          case 'show':
+            elements.each(() => {
+              const stepTitle = $(this)
+                .parents('.step__title')
+                .find('a')
+                .text();
+              $(this)
+                .parents('.step')
+                .find('.step__summary')
+                .addClass('step-show-summary');
+              $(this).text(stepByStep.hideStepText);
+              $(this).attr('aria-expanded', 'true');
+              $(this).attr(
+                'aria-label',
+                Drupal.t('Hide step summary - !summary_message', {
+                  '!summary_message': stepTitle,
+                }),
+              );
+            });
+            // 'Hide all' control displayed if all steps are shown.
+            if ($('.step__summary').length === $('.step-show-summary').length) {
+              $('.step-master').text(stepByStep.hideAllText);
+              $('.summaries-control i')
+                .addClass('fa-eye-slash')
+                .removeClass('fa-eye');
+            }
+            break;
 
-  // Set visibility based on specified button.step-show elements.
-  function summaryVisiblity(elements, cmd) {
-    switch(cmd) {
-      case 'show':
-        elements.each(function() {
-          var stepTitle = $(this).parents('.step__title').find('a').text();
-          $(this).parents('.step').find('.step__summary').addClass('step-show-summary');
-          $(this).text(stepByStep.hideStepText);
-          $(this).attr("aria-expanded", "true");
-          $(this).attr('aria-label', Drupal.t("Hide step summary - !summary_message", {"!summary_message": stepTitle}));
-        });
-        // 'Hide all' control displayed if all steps are shown.
-        if ($('.step__summary').length === $('.step-show-summary').length) {
-          $('.step-master').text(stepByStep.hideAllText);
-          $('.summaries-control i').addClass('fa-eye-slash').removeClass('fa-eye');
+          case 'hide':
+            elements.each(() => {
+              const stepTitle = $(this)
+                .parents('.step__title')
+                .find('a')
+                .text();
+              $(this)
+                .parents('.step')
+                .find('.step__summary')
+                .removeClass('step-show-summary');
+              $(this).attr('aria-expanded', 'false');
+              $(this).text(stepByStep.showStepText);
+              $(this).attr(
+                'aria-label',
+                Drupal.t('Show step summary - !summary_message', {
+                  '!summary_message': stepTitle,
+                }),
+              );
+            });
+            // 'Show all' control displayed if any steps are hidden.
+            $('.step-master').text(stepByStep.showAllText);
+            $('.summaries-control i')
+              .addClass('fa-eye')
+              .removeClass('fa-eye-slash');
+            break;
+          default:
+            break;
         }
-        break;
+      }
 
-      case 'hide':
-        elements.each(function() {
-          var stepTitle = $(this).parents('.step__title').find('a').text();
-          $(this).parents('.step').find('.step__summary').removeClass('step-show-summary');
-          $(this).attr("aria-expanded", "false");
-          $(this).text(stepByStep.showStepText);
-          $(this).attr('aria-label',  Drupal.t("Show step summary - !summary_message", {"!summary_message": stepTitle}));
-        });
-        // 'Show all' control displayed if any steps are hidden.
-        $('.step-master').text(stepByStep.showAllText);
-        $('.summaries-control i').addClass('fa-eye').removeClass('fa-eye-slash');
-        break;
-    }
-  }
+      // Insert show all button.
+      $(
+        `<div class='summaries-control'><i class='fas fa-eye'></i><button aria-expanded='false' class='step-master ml-2'>${stepByStep.showAllText}</button></div>`,
+      ).insertBefore('ol.step-list');
 
-  // Insert show all button.
-  $("<div class='summaries-control'><i class='fas fa-eye'></i><button aria-expanded='false' class='step-master ml-2'>" + stepByStep.showAllText + "</button></div>").insertBefore("ol.step-list");
+      // Insert hide/show button for each step.
+      function stepSummaryButton(isVisible, stepTitle) {
+        const $container = $("<span class='step-summary-container'>");
+        const $button = $("<button class='step-show'>");
+        $button.attr('aria-expanded', isVisible ? 'true' : 'false');
+        if (isVisible) {
+          $button.attr(
+            'aria-label',
+            Drupal.t('Hide step summary - !summary_message', {
+              '!summary_message': stepTitle,
+            }),
+          );
+        } else {
+          $button.attr(
+            'aria-label',
+            Drupal.t('Show step summary - !summary_message', {
+              '!summary_message': stepTitle,
+            }),
+          );
+        }
+        $button.text(
+          isVisible ? stepByStep.hideStepText : stepByStep.showStepText,
+        );
+        $container.append($button);
+        return $container;
+      }
 
-  // Insert hide/show button for each step.
-  function stepSummaryButton(isVisible, stepTitle) {
-    var $container = $("<span class='step-summary-container'>");
-    var $button = $("<button class='step-show'>");
-    $button.attr('aria-expanded', isVisible ? "true" : "false");
-    if (isVisible) {
-      $button.attr('aria-label', Drupal.t("Hide step summary - !summary_message", {"!summary_message": stepTitle}));
-    } else {
-      $button.attr('aria-label', Drupal.t("Show step summary - !summary_message", {"!summary_message": stepTitle}));
-    }
-    $button.text(isVisible ? stepByStep.hideStepText : stepByStep.showStepText);
-    $container.append($button);
-    return $container;
-  }
+      $('ol.step-list .step').each(() => {
+        const isVisible = $(this).hasClass('step--active');
+        const stepTitle = $(this).find('.step__title').text();
+        if (isVisible) {
+          $(this).find('.step__summary').addClass('step-show-summary');
+        }
+        $(this)
+          .find('.step__title')
+          .append(stepSummaryButton(isVisible, stepTitle));
+      });
 
-  $("ol.step-list .step").each(function() {
-    var isVisible = $(this).hasClass('step--active');
-    var stepTitle = $(this).find('.step__title').text();
-    if (isVisible) {
-      $(this).find('.step__summary').addClass('step-show-summary');
-    }
-    $(this).find('.step__title').append(stepSummaryButton(isVisible, stepTitle));
-  });
+      // Show / hide all.
+      $('.step-master').on('click', () => {
+        $('.summaries-control i').toggleClass('fa-eye fa-eye-slash');
+        if ($(this).text() === stepByStep.showAllText) {
+          $(this).text(stepByStep.hideAllText).attr('aria-expanded', true);
+          summaryVisiblity($('.step-show'), 'show');
+        } else {
+          $(this).text(stepByStep.showAllText).attr('aria-expanded', false);
+          summaryVisiblity($('.step-show'), 'hide');
+        }
+      });
 
-  // Show / hide all.
-  $('.step-master').on("click", function () {
-    $('.summaries-control i').toggleClass('fa-eye fa-eye-slash');
-    if ($(this).text() === stepByStep.showAllText) {
-      $(this).text(stepByStep.hideAllText).attr('aria-expanded', true);
-      summaryVisiblity($('.step-show'), 'show');
-    } else {
-      $(this).text(stepByStep.showAllText).attr('aria-expanded', false);
-      summaryVisiblity($('.step-show'), 'hide');
-    }
-  });
-
-  // Show / hide single step.
-  $('.step-show').on("click", function () {
-    $(this).parents('.step').find('.step__summary').toggleClass('step-show-summary');
-    if ($(this).text() === stepByStep.showStepText) {
-      summaryVisiblity($(this), 'show');
-    } else {
-      summaryVisiblity($(this), 'hide');
-    }
-  });
-
-})(jQuery, Drupal, drupalSettings);
+      // Show / hide single step.
+      $('.step-show').on('click', () => {
+        $(this)
+          .parents('.step')
+          .find('.step__summary')
+          .toggleClass('step-show-summary');
+        if ($(this).text() === stepByStep.showStepText) {
+          summaryVisiblity($(this), 'show');
+        } else {
+          summaryVisiblity($(this), 'hide');
+        }
+      });
+    },
+  };
+})(jQuery, Drupal);

--- a/js/step-by-step-nav.js
+++ b/js/step-by-step-nav.js
@@ -5,7 +5,7 @@
 
 (function lgdStepByStepScript($, Drupal) {
   Drupal.behaviors.lgdStepByStepNav = {
-    attach(context) {
+    attach() {
       const stepByStep = {};
       stepByStep.showAllText = 'Show summaries';
       stepByStep.hideAllText = 'Hide summaries';
@@ -16,7 +16,7 @@
       function summaryVisiblity(elements, cmd) {
         switch (cmd) {
           case 'show':
-            elements.each(() => {
+            elements.each(function showSummary() {
               const stepTitle = $(this)
                 .parents('.step__title')
                 .find('a')
@@ -44,7 +44,7 @@
             break;
 
           case 'hide':
-            elements.each(() => {
+            elements.each(function hideSummary() {
               const stepTitle = $(this)
                 .parents('.step__title')
                 .find('a')
@@ -68,6 +68,7 @@
               .addClass('fa-eye')
               .removeClass('fa-eye-slash');
             break;
+
           default:
             break;
         }
@@ -75,7 +76,9 @@
 
       // Insert show all button.
       $(
-        `<div class='summaries-control'><i class='fas fa-eye'></i><button aria-expanded='false' class='step-master ml-2'>${stepByStep.showAllText}</button></div>`,
+        `<div class='summaries-control'><i class='fas fa-eye'></i><button aria-expanded='false' class='step-master ml-2'>${
+          stepByStep.showAllText
+        }</button></div>`,
       ).insertBefore('ol.step-list');
 
       // Insert hide/show button for each step.
@@ -105,7 +108,7 @@
         return $container;
       }
 
-      $('ol.step-list .step').each(() => {
+      $('ol.step-list .step').each(function initializeStep() {
         const isVisible = $(this).hasClass('step--active');
         const stepTitle = $(this).find('.step__title').text();
         if (isVisible) {
@@ -117,7 +120,7 @@
       });
 
       // Show / hide all.
-      $('.step-master').on('click', () => {
+      $('.step-master').on('click', function toggleAllSteps() {
         $('.summaries-control i').toggleClass('fa-eye fa-eye-slash');
         if ($(this).text() === stepByStep.showAllText) {
           $(this).text(stepByStep.hideAllText).attr('aria-expanded', true);
@@ -129,7 +132,7 @@
       });
 
       // Show / hide single step.
-      $('.step-show').on('click', () => {
+      $('.step-show').on('click', function toggleSingleStep() {
         $(this)
           .parents('.step')
           .find('.step__summary')
@@ -142,4 +145,4 @@
       });
     },
   };
-})(jQuery, Drupal);
+})(jQuery, Drupal, drupalSettings);

